cfi_intro.adoc

```diff
@@ -128,6 +108,27 @@ for indirect calls and jumps in a program. Compiler is expected to emit a `lpcll
 as the first instruction of address-taken functions. Compiler is expected to
 emit a `lpcll` at an indirect jump target.
 
+The Zisslpcfi extension identifies an indirect call or an indirect jump as
+follows:
+
+[[IND_CALL_JMP]]
+.Indirect call/jump determination
+[source, text]
+----
+    if ( ((JALR && rd == x0) || (C.JR)) && (rs1 != x1) && (rs1 != x5) )
+        is_indirect_jump = 1;
+    else
+        is_indirect_jump = 0;
+
+    if ( (JALR && rd != x0) || (C.JALR) )
```

Comment:
It's really `rd != x0` rather than checking for `x1` or `x5`?

Reply:
Yes,